### PR TITLE
feat: custom matcher toBeChecked

### PR DESCRIPTION
**Pull request by cexbrayat**
On top of #13

### CODE REVIEW

File: projects/ngx-speculoos/src/lib/matchers.spec.ts [hunk 1]
```diff
@@ -212,4 +218,123 @@ describe('Custom matchers', () => {
...
+  });
+
+  describe('toBeUnchecked', () => {
+    const matcher = speculoosMatchers.toBeUnchecked(undefined, undefined);
+
+    it('should check if unchecked', () => {
+      tester.checkbox.uncheck();
+      expect(tester.checkbox).toBeUnchecked();
+      tester.checkbox.check();
+      expect(tester.checkbox).not.toBeUnchecked();
```
Comment (jnizet) [inline projects/ngx-speculoos/src/lib/matchers.spec.ts]: The first actual test I see doing that, I murder the programmer :-)

File: projects/ngx-speculoos/src/lib/matchers.spec.ts [hunk 2]
```diff
@@ -212,4 +218,123 @@ describe('Custom matchers', () => {
...
+      expect(result.pass).toBeTruthy();
+    });
+
+    it('should return false if checked', () => {
+      const result = matcher.compare(tester.checkbox);
+      expect(result.pass).toBeFalsy();
+      expect(result.message).toBe(`Expected element to be unchecked, but was not`);
+    });
+
+    it('should return false if checked and .not', () => {
```
Comment (jnizet) [inline projects/ngx-speculoos/src/lib/matchers.spec.ts]: should return true

File: projects/ngx-speculoos/src/lib/matchers.spec.ts [hunk 3]
```diff
@@ -212,4 +218,123 @@ describe('Custom matchers', () => {
...
+      const result = matcher.negativeCompare(tester.checkbox);
+      expect(result.pass).toBeTruthy();
+    });
+
+    it('should return true if unchecked and .not', () => {
```
Comment (jnizet) [inline projects/ngx-speculoos/src/lib/matchers.spec.ts]: should return false

File: projects/ngx-speculoos/src/lib/matchers.spec.ts (hunk 2 above)
Comment (cexbrayat) [reply projects/ngx-speculoos/src/lib/matchers.spec.ts]: Good catch

**Comment by cexbrayat** [comment]
Fixed. Waiting on approval to merge

**Pull request closed without merging**